distance_based_on_real_case.py

```diff
@@ -0,0 +1,1133 @@
+)
+
+SEED = 42
+ALPHA = 1
+NLAYER_PER_BATCH = (
+    40  # 66 layers in ERTBOX for Drogon which is split into 2 batches here
+)
+# Turn on/off calculation of rho based on localization or only use rho = 1
+USE_LOCALIZATION = False
+
+# Turn on/off export of field parameters into 3D parameter files in ROFF format to be read by RMS.
+WRITE_STATS_PARAMS = True
+
+# Common definition of ERTBOX for all zones in Drogon
+XORIGO = 456063.69091796875
+YORIGO = 5935990.898925781
+XINC = 75.17192661654963
+YINC = 74.79699019286826
+ROTATION = 29.88159640627248
+NX = 92
+NY = 146
+NZ = 66
+
+# Need to distinguish between observation groups.
+# NOTE: This should be available from ERT, but is not yet?
+# WellObs - Observations from wells with position (prod data etc)"
+# GenObs  - Any observation having a location like seismic obs points,
+#           gravimetry obs points
+# RegionObs - Any observation without any specific coordinate for
+#             its location (e.g. region or field prod data,
+#             data for average over regions, FWL, GOC which are
+#             defined per region)
+OBS_TYPES_WITH_LOCATION = ["WellObs", "GenObs"]
+OBS_TYPES_WITHOUT_LOCATION = ["RegionObs"]
+
+# Get observations with position and localization ranges from external files
+# created by running a script in RMS
+OBS_FILE_PATH = (
+    "/project/fmu/users/olia/drogon_20250623_11-50/resmod/ff/25.0.0/rms/model/"
+)
+OBS_FILE = "summary_obs_with_localization_attributes.csv"
+OBS_SUMMARY_WITH_LOCAL_ATTRIBUTES_FILE = OBS_FILE_PATH + OBS_FILE
+
+# Hard-coded a classification of parameter groups into scalars, 2D or 3D field parameters
+# NOTE: This should be available from ERT storage (ERT knows about the source the keyword FIELD, SURFACE GEN_KW)
+#       and could have an attribute in storage for the type of the parameter group.
+GROUP_WITH_3D_FIELDS = [
+        "aps_Valysar_GRF1",
+        "aps_Valysar_GRF2",
+        "aps_Valysar_GRF3",
+        "aps_Therys_GRF1",
+        "aps_Therys_GRF2",
+        "aps_Therys_GRF3",
+        "aps_Volon_GRF1",
+        "aps_Volon_GRF2",
+        "aps_Volon_GRF3",
+    ]
+GROUP_WITH_2D_FIELDS = [
+        "TOP_VOLANTIS",
+        "TOP_THERYS",
+        "TOP_VOLON",
+        "BASE_VOLANTIS",
+    ]
+GROUP_WIHT_SCALARS = [
+        "GLOBVAR",
+        "MULTREGT",
+        "RATE",
+    ]
+
+# If we could have naming convention we could get zone name from parameter name.
+# We have that for APS parameters:  aps_<zone_name>_<grf_name>),
+# In general we don't. A proposal for naming convention for petrophysical field parameters
+# (e.g for permeabilit or porosity) could be <zone_name>_<facies>_<petro_name>
+# for petrophysical fields to be updated simultaneously with facies update from APS.
+# For reservoir zones without facies modelling, the convention could be  <zone_name>_<petro_name>
+# NOTE: An additional parameter in FIELD keyword with zone name can solve the problem if naming convention
```

Comment:
An issue has been created on this that needs more information:

https://github.com/equinor/ert/issues/11517